BuildAddOn.py

```diff
@@ -223,13 +228,13 @@ def BuildAddOns (args, configData, platformName, languageList, workspaceRootFold
         for version in devKitFolderList:
             devKitFolder = devKitFolderList[version]
 
-            if args.release is True:
+            if args.allLocalizedVersions is True:

```

Comment:
Why do we need to check `allLocalizedVersions` here? The `languageList` will contain a the default language only if it's not set to true.

Reply:
Because localized builds are made with only RelWithDebInfo configuration, since they represent the release builds.

Reply:
These are two different things. One tells which languages should be built, the other is the build configuration. I think it's ok to build only `RelWithDebInfo` if `package` is true, but it doesn't have anything to do with the languages.

Reply:
Fixed